spec/models/efile_submission_spec.rb

```diff
@@ -175,6 +175,27 @@
       end
     end
 
+
+    context "fraud_hold" do
+      let(:submission) { create :efile_submission, :fraud_hold }
+      transitionable_states = [:preparing, :queued, :failed, :rejected, :resubmitted, :investigating, :waiting]
+      context "can transition to " do
+        transitionable_states.each do |state|
+          it state.to_s do
+            expect { submission.transition_to!(state) }.not_to raise_error
+          end
+        end
+      end
+
+      context "cannot transition to" do
+        EfileSubmissionStateMachine.states.excluding(*transitionable_states.map(&:to_s)).each do |state|
```

Comment:
In the story it says submission can transition from fraud hold to any status?

Reply:
yes, but it ended up being too broad considering the functionality of all of the statuses.